LetsDinnerV2 MessagesExtension/Controller/RegistrationVC/RegistrationViewController.swift

```diff
@@ -537,6 +523,7 @@ class RegistrationViewController: LDNavigationViewController {
             make.top.equalToSuperview().offset(10)
             make.width.equalTo(userPic.snp.height)
             make.bottom.equalTo(addPicButton.snp.top).offset(-10)
+            make.centerY.equalToSuperview()
```

Comment:
Why? There is already a top and a bottom constraint

Reply:
mistake